clippy_lints/src/lib.rs

```diff
@@ -12,7 +12,6 @@
 #![cfg_attr(feature = "deny-warnings", deny(warnings))]
 #![feature(crate_visibility_modifier)]
 #![feature(concat_idents)]
-#![feature(result_map_or)]
```

Comment:
This been removed and merged into master in #4907. Just a heads up.

Reply:
Thanks for the tip. Let me rebase again.